apps/email/style/compose_cards.css

```diff
@@ -22,6 +22,8 @@
 .cmp-bubble-container {
   width: calc(100% - 1rem);
   float: left;
+  display: inline-flex;
+  flex-wrap: wrap;
```

Comment:
This was actually for the input field. I can see what i can do there with the old styles (but also see if something can be done with the above styles too)